src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from "react";
+import * as FontAwesome from "react-fontawesome";
+import * as cx from "classnames";
+
+import { FlexBox } from "../flex";
+// import LoadingSpinner from "../LoadingSpinner";
+
+import "./MessagingAttachment.less";
+
+function cssClass(element: string) {
+  return `MessagingAttachment--${element}`;
+}
+
+type Props = {
+  attachmentID: string;
+  errorMsg?: string;
+  icon: React.ReactNode;
+  onClickAttachment: (attachmentID: string) => void;
+  onRemoveAttachment?: (attachmentID: string) => void;
+  title: string;
+  subtitle?: string;
+  isUpload?: boolean;
+  uploadComplete?: boolean;
+  uploadError?: boolean;
+};
+
+// TODO: can this not be functional?
```

Comment:
Not sure I understand this TODO? It's already a functional component, after all. 😮

Reply:
I guess I meant just the different way to write it, using `function` but you're right